src/types/base-types.ts

```diff
@@ -0,0 +1,46 @@
+import { asNumber, asObject, asString } from 'cleaners'
+
+import { normalizePath } from '../util/paths'
+
+// Regexes:
+export const VALID_PATH_REGEX = /^[a-z0-9-_.]+(\/+[a-z0-9-_.]+)*$/i
```

Comment:
Do we want to export these regexes in the public library API? Maybe these could be moved to the `util/paths.ts` file, so the unit tests and cleaners could still access them, but they would no longer be exported from this public types file.